pcsx2/Achievements.h

```diff
@@ -250,10 +263,16 @@ namespace Achievements
 	{
 	}
 
+	static std::string GetRichPresenceString()
```

Comment:
If we don't drop the option, this is going to cause issues - it's returning a const reference in the enable path, and I'm fairly sure there's places where I did ```cpp
const std::string& rp = Achievements::GetRichPresenceString();
```

which would mean taking the address of a temporary. No real clean way around that one...